lib/extensions/ar_virtual.rb

```diff
@@ -209,6 +209,9 @@ def load_schema!
       super
 
       virtual_attributes_to_define.each do |name, (type, options)|
+        if type.respond_to?(:call)
+          type = type.call
+        end
```

Comment:
should the following line be `elsif` or can the `type` after the call be a symbol and thus processed in next `if`?